doc/api/training/smd_model_parallel.rst

```diff
@@ -1,30 +1,30 @@
 Distributed model parallel
 --------------------------
 
-Amazon SageMaker Distributed Model Parallel (SMP) is a model parallelism library for training
+The Amazon SageMaker distributed model parallel library is a model parallelism library for training
```

Comment:
Instead of SDP we say "Sagemaker's distributed data parallel library" and for SMP we say "Sagemaker distributed model parallel library". Maybe it should be uniform ?

Reply:
It reads better this way when there's an "Amazon" in front. I think the spirit of the update is to make sure they're lower case and try to make it sound somewhat normal with that mouthful of words.

Reply:
I think we can use either route when we want. "SageMaker's xxx" or "The SageMaker xxx"...